en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md

```diff
@@ -43,7 +43,7 @@ Ex: Enable landing page. **defaultTheme.json** ( Teant theming ).
 
 **If you do not have access to the file system** , you can upload the Devportal theme via the Admin Portal as shown below:
 
-1.  Download the sample theme here [sampleTheme.zip]({{base_path}}/assets/attachments/learn/sampleTheme.zip). 
+1.  Download the sample theme here [sampleTheme.zip]({{base_path}}/assets/attachments/learn/SampleTheme.zip). 
```

Comment:
This SampleTheme.zip file seems to be broken. I upload a corrected zip with name sample-theme.zip.

Reply:
https://github.com/wso2/docs-apim/pull/1154

Reply:
The change has been done,